scripts/copyFilesUtils.mjs

```diff
@@ -17,10 +17,17 @@ const buildPath = path.join(packagePath, './build');
 export async function includeFileInBuild(file, target = path.basename(file)) {
   const sourcePath = path.resolve(packagePath, file);
   const targetPath = path.resolve(buildPath, target);
-  await fse.copy(sourcePath, targetPath);
+  await fs.copyFile(sourcePath, targetPath);
   console.log(`Copied ${sourcePath} to ${targetPath}`);
 }
 
+function pathExists(pathToTest) {
+  return fs
+    .stat(pathToTest)
+    .then(() => true)
```

Comment:
```suggestion
    .then((stat) => stat.isFile())
```

Reply:
I'm not sure we can do this. `cjsCopy` relies on `pathExists` and provides a path to a directory, so I think this change would cause a change in behavior.